login-workflow/src/new-architecture/contexts/RegistrationContext/provider.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * @packageDocumentation
+ * @module RegistrationWorkflowContextProvider
+ */
+
+import React, { useEffect } from 'react';
+import { RegistrationContextProviderProps } from './types';
+import { RegistrationContext } from './context';
+import { I18nextProvider, useTranslation } from 'react-i18next';
+import { i18nRegistrationInstance } from './i18nRegistrationInstance';
+import Typography from '@mui/material/Typography';
+
+export const RegistrationContextProvider: React.FC<React.PropsWithChildren<RegistrationContextProviderProps>> = (
+    props
+) => {
+    const { language, i18n = i18nRegistrationInstance } = props;
+    const { t } = useTranslation();
```

Comment:
In theory, this shouldn't work because your haven't rendered the i18n provider yet, so this hook won't see it....it will go up the tree until it finds the global provider...but you're only using this for your placeholder items below which are going to be removed anyway.